com.unity.render-pipelines.high-definition/Editor/Wizard/HDRPWizard.Configuration.cs

```diff
@@ -29,7 +28,7 @@ public static bool Contains(this InclusiveScope thisScope, InclusiveScope scope)
             => ((~thisScope) & scope) == 0;
     }
 
-    partial class HDWizard
+    partial class HDRPWizard
```

Comment:
Is this necessary?
Up to now we used Universal on one side and HD on the other side.
Example: HDAdditionalCameraData which is public and would be almost impossible to rename without impacts on users.

So for alignment if we change HD to HDRP we should change Universal to UniversalRP, no ?

@MaddalenaUnity @TheoWong-pixel , should we do this?

Reply:
(I mean the class name, not the actual menu title)

Reply:
Here I think it makes sense to mantain the class name with the window name, as this Wizard if for all HDRP, and it clarifies that is not a Wizard for just High Definition. I am not sure If I would change everything for users, and I will short UniversalRP to UPR.

Reply:
Post pone the class name change for now. This need to be discussed fully to uniformize namings.